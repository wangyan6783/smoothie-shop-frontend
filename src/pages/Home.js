import React, { Component } from 'react';
import Hero from '../components/Hero';

class Home extends Component {
    render(){
        return (
            <Hero />
        )
    }
}

export default Home;