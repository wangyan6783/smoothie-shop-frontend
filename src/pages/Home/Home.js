import React from "react";
import { Link } from "react-router-dom";
import homeImage from "../../assets/home.png";

function Home() {
  return (
    <div className="home">
      <div className="home-content">
        <h1 className="heading-1 home-title mb-sm">Eat more fruites</h1>
        <p className="content-text-1 mb-md">
          We take care of food so food can take care of you.
        </p>
        <Link to="/smoothies">
          <button className="btn home-btn">Get started</button>
        </Link>
      </div>
      <img className="home-image" src={homeImage} alt="Eat more fruites" />
    </div>
  );
}

export default Home;
