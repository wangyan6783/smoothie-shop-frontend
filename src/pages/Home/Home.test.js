import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import renderer from "react-test-renderer";
import Home from "./Home";

describe("Home Component", () => {
  it("render correctly", () => {
    const homeComponent = renderer
      .create(
        <Router>
          <Home />
        </Router>
      )
      .toJSON();
    expect(homeComponent).toMatchSnapshot();
  });
});
