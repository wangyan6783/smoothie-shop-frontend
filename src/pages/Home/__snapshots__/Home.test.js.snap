// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component render correctly 1`] = `
<div
  className="home"
>
  <div
    className="home-content"
  >
    <h1
      className="heading-1 home-title mb-sm"
    >
      Eat more fruites
    </h1>
    <p
      className="content-text-1 mb-md"
    >
      We take care of food so food can take care of you.
    </p>
    <a
      href="/smoothies"
      onClick={[Function]}
    >
      <button
        className="btn home-btn"
      >
        Get started
      </button>
    </a>
  </div>
  <img
    alt="Eat more fruites"
    className="home-image"
    src="home.png"
  />
</div>
`;
